src/OrbitClient.php

```diff
@@ -201,6 +201,8 @@ private function getRequestHeaders(array $options)
         }
         if ($this->options['env'] === 'stage') {
             $headers['x-forge-environment'] = 'stage';
+        }
+        if (in_array($this->options['env'], ['int', 'stage', 'test'])) {
```

Comment:
do we want this in the sandbox?

Reply:
While I don't think it would matter to Orbit, maybe its prudent to put it in incase we have to use it/know about it in future. I can add 'sandbox' to the array now